src/Wrap.js

```diff
@@ -1,10 +1,14 @@
 // noinspection ES6UnusedImports
 import { h } from 'preact';
-import uuid from 'uuid';
+
+const randomId = () =>
```

Comment:
Cool, I would just rather have this function isolated somewhere else.
Maybe a level up? Just to avoid interfering with Wrap responsibilities.